CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 
+- **[NEW]** Add info button props to `Item` and `ItemData`
```

Comment:
nit: Add `leftTitleButtonAddon` property to `ItemData`

Item isn't readlly exposed, so no need to talk about it here.